lib/job.js

```diff
@@ -193,9 +196,15 @@ Job.prototype.run = function(cb) {
 };
 
 Job.prototype.isRunning = function() {
-  if (!this.attrs.lastRunAt) return false;
-  if (!this.attrs.lastFinishedAt) return true;
-  if (this.attrs.lockedAt && this.attrs.lastRunAt.getTime() > this.attrs.lastFinishedAt.getTime()) return true;
+  if (!this.attrs.lastRunAt) {
```

Comment:
Keep the first two as one liners since they're so easy to read. The 3rd one can stay as a multi-liner as you did it.